theseus/geometry/so3.py

```diff
@@ -12,6 +7,11 @@
 from .lie_group import LieGroup
 from .point_types import Point3
 
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
This should be at the top of the file.

Reply:
This issue is fixed in the newest commit.